### DISCUSSION TITLE
What is purpose of validating user?

### DISCUSSION THREAD

**Discussion by beykansen** [umputun/remark42#1731] [Q&A] [answered]
Hi. I don't understand user validation feature. What is it for?

![CleanShot 2024-01-25 at 20 38 13@2x](https://github.com/umputun/remark42/assets/7861440/b93f6996-ec68-4a67-93eb-396f5c83e237)

**Comment by umputun** [comment, marked as answer]
For whatever you want it to represent. Can be an indication of a privileged user, for example

**Reply by beykansen** [reply]
Thank you for your work. 🙏
